apps/i18n/common/en_us.json

```diff
@@ -1096,6 +1100,7 @@
   "goToMyDashboard": "Go to my Dashboard",
   "goToUnit": "Go to Unit",
   "grade": "Grade",
+  "gradeLevel": "Grade {level}",
```

Comment:
nit –– I think the variable being called {number} rather than {level} might help the translators. "Level" to me implies something like "5th" rather than "5"

Reply:
Oh good point! I'll fix that